dashboard/app/views/layouts/_header.html.haml

```diff
@@ -98,7 +98,7 @@
                 = render file: Rails.root.join('..', 'shared', 'haml', 'hamburger.haml'), locals: hamburger_options
 
 - if show_header2
-  - if request.path == "/" || request.path == "/courses"
+  - if request.path == "/home" || request.path == "/courses"
```

Comment:
The parent view is here: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/layouts/application.html.haml#L37

And didn't seem especially better.

Ideally we'd have something like Pegasus' page properties (https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/index.haml#L1-L14) which surrounding components (like headers and footers) can query as the page is being built, but I'm not sure whether dashboard has an equivalent?